### ISSUE TITLE
.setPricacyStatus does not work on ios

### ISSUE THREAD

**Issue by HelgeWieding** [adobe/cordova-acpcore#9]
### Expected Behaviour
setPrivacyStatus should set the status according to the value that is passed to it. 
### Actual Behaviour
privacyStatus is always set to 2 (Unknown)
### Reproduce Scenario (including but not limited to)

when i call ```ACPCore.setPrivacyStatus(ACPCore.ACPMobilePrivacyStatusOptIn, this.trackingSuccess, this.trackingError);```
in my service and directly after that call ```ACPCore.setPrivacyStatus.getPrivacyStatus(....``` it returns 2; 

My ACPCore set-up seems to be correct, it calls the function of the Plugin with the correct value. 
If i call ACPCore.setPrivacyStatus directly from swift code it also works. And events are being sent. 

#### Steps to Reproduce
call ACPCore.setPrivacyStatus(ACPCore.ACPMobilePrivacyStatusOptIn ...
see in xcode that reqauests are being queued

#### Platform and Version
ios 12.1  
#### Sample Code that illustrates the problem
```
 allowTrackingChange(allow: boolean) {
    if (!allow) {
      ACPCore.setPrivacyStatus(ACPCore.ACPMobilePrivacyStatusOptOut, this.trackingSuccess, this.trackingError);
    } else {
      ACPCore.setPrivacyStatus(ACPCore.ACPMobilePrivacyStatusOptIn, this.trackingSuccess, this.trackingError);
    }
    
    ACPCore.getPrivacyStatus(status => console.log(status)) // returns 2
  }
```

#### Logs taken while reproducing problem
![logs](https://user-images.githubusercontent.com/754426/126319657-ec6dd7a1-bd43-435c-b611-22c1d9513a61.png)

**Comment by rymorale** [comment]
Hi @HelgeWieding, 
Thanks for opening the issue. It has been fixed in the cordova-acpcore 1.0.0 release. Please let us know if you see any other issues.
Thanks,
Ryan

**Comment by HelgeWieding** [comment]
Thanks for the fix!
